pkgs/top-level/all-packages.nix

```diff
@@ -29649,6 +29649,30 @@ with pkgs;
 
   shepherd = nodePackages."@nerdwallet/shepherd";
 
+  inherit (callPackage ../applications/virtualization/singularity/packages.nix { })
+    apptainer
+    singularity;
+
+  _apptainer-nixos-overriden-default = (nixos {
+    programs.singularity = {
+      enable = true;
+      package = apptainer;
+    };
+  }).config.programs.singularity.packageOverriden
+  .overrideAttrs (oldAttrs: {
+    meta.description = "";
+  });
+
+  _singularity-nixos-overriden-default = (nixos {
+    programs.singularity = {
+      enable = true;
+      package = singularity;
+    };
+  }).config.programs.singularity.packageOverriden
+  .overrideAttrs (oldAttrs: {
+    meta.description = "";
+  });
```

Comment:
Please move this into the file we already inherit from. Also Can we give this a nice name even if it is internal?